src/main/java/uk/gov/hmcts/reform/em/hrs/model/CaseRecordingFile.java

```diff
@@ -18,7 +18,7 @@
     private CaseDocument caseDocument;
 
     @JsonProperty("segmentNumber")
-    private Integer segmentNumber;
+    private String segmentNumber;
```

Comment:
is this part of the revert?

Reply:
no, but we've got a bug where segmentNumber doesn't show up if it's 0. I'm assuming it's lost when it get serialised, so changing it to string to avoid that, given that it's not really needed as number on CCD